apps/web/src/design-system/ThemeProvider.tsx

```diff
@@ -114,6 +114,7 @@ export function ThemeProvider({ children }: { children: JSX.Element; dark?: Bool
               ...theme.fn.fontStyles(),
               backgroundColor: theme.colorScheme === 'dark' ? colors.BGDark : colors.BGLight,
               color: theme.colorScheme === 'dark' ? colors.white : colors.B40,
+              marginRight: `calc(-1 * var(--removed-scroll-width, 0))`,
```

Comment:
when the drawer is opened on a scrollable page it removes the scroll... and we had a bug where the body was not positioned properly and because of that the header had a gap on the right side where the scrollbar was
![Screenshot 2023-07-07 at 12 15 52](https://github.com/novuhq/novu/assets/2607232/36b6b483-80b4-4a72-b376-a78397b96f86)